ros_alarms/alarms.py

```diff
@@ -113,13 +114,18 @@ def get_alarm(self):
 
         params = resp.alarm.parameters
         resp.alarm.parameters = params if params == '' else json.loads(params)
+        _last_alarm = resp.alarm
         return resp.alarm 
 
     def _severity_cb_check(self, severity):
+        # In case _last alarm hasnt been declared yet
+        if not '_last_alarm' in locals():
```

Comment:
Why is it weird? It's just checking for it in the list of local variables. I was meaning to take it out of the init since you said it was probably best to not let an alarm be None.